### PR TITLE
[tools/dv] Remove set_fsm_reset_scoring coverage directive

### PR DESCRIPTION
**Pull request by senelson7**
Removed "set_fsm_reset_scoring" from common.ccf file.

Signed-off-by: Steve Nelson <steve.nelson@wdc.com>

### CODE REVIEW

**Comment by senelson7** [comment]
The only transitions that aren't currently covered are from all states to idle due to reset. @weicaiyang you had previously indicated I could waive this for V2 if it's only due to reset. I am working with Cadence FAE because I feel I am verifying but for now I'd like to comment-out, I can add a TODO comment. Anyways, you indicated you have this enabled for vcs. Are other blocks verifying reset path?

**Comment by weicaiyang** [comment]
> The only transitions that aren't currently covered are from all states to idle due to reset. @weicaiyang you had previously indicated I could waive this for V2 if it's only due to reset. 

I think we do want to verify transition from all states to idle. The stress_all_with_rand_reset may help verify some of these transition. But it's not that high priority so that it's OK to do it after V2.

Add @sriyerg 

> I am working with Cadence FAE because I feel I am verifying but for now I'd like to comment-out, I can add a TODO comment

The way of waiving FSM coverage is to add a comment in the checklist to indicate that the only missing transitions are from other states to reset. The waive should be added case by case. In some blocks, the FSM has 90%+, which doesn't need to be waived.
I'm not sure why you comment this out. This will apply to all blocks signed off with Xcelium. Is the FSM scoring wrong?

**Comment by senelson7** [comment]
Maybe, will continue to work with Cadence. Seems like I'm verifying but tool still says I'm not.

**Comment by weicaiyang** [comment]
> Maybe, will continue to work with Cadence. Seems like I'm verifying but tool still says I'm not.

Sounds like a tool issue or we didn't use it correctly. Perhaps add a TODO and file an issue to keep track of this if you want to merge this.

**Comment by mwbranstad** [comment]
@senelson7 based on the debate, is this line to be kept or removed. maybe still a WIP.

**Comment by senelson7** [comment]
We'll keep it in until I've worked with Cadence some more

File: hw/dv/tools/xcelium/common.ccf [hunk 1]
```diff
@@ -34,7 +34,7 @@ set_statement_scoring
 set_toggle_scoring -sv_enum enable_mda -sv_struct_with_enum -sv_modport -sv_mda 16 -sv_mda_of_struct -sv_generate -sv_packed_union
 
 // Enables scoring of reset states and transitions for identified FSMs.
-set_fsm_reset_scoring
+// set_fsm_reset_scoring
```
Comment (weicaiyang) [inline hw/dv/tools/xcelium/common.ccf:37]: nit: perhaps remove these lines, we probably don't need them any more since Lint will cover this.

**Review by weicaiyang — APPROVED** [review]
LGTM with a nit

**Pull request merged**